modules/nf-core/msisensorpro/msitumoronly/meta.yml

```diff
@@ -0,0 +1,68 @@
+name: msisensorpro_msitumoronly
+description: MSIsensor-pro evaluates Microsatellite Instability (MSI) for cancer patients with next generation sequencing data. It accepts the whole genome sequencing, whole exome sequencing and target region (panel) sequencing data as input
+keywords:
+  - micro-satellite-scan
+  - msisensor-pro
+  - msi
+  - somatic
+  - tumor-only
+tools:
+  - msisensorpro:
+      description: Microsatellite Instability (MSI) detection using high-throughput sequencing data.
+      homepage: https://github.com/xjtu-omics/msisensor-pro
+      documentation: https://github.com/xjtu-omics/msisensor-pro/wiki
+      tool_dev_url: https://github.com/xjtu-omics/msisensor-pro
+      doi: "10.1016/j.gpb.2020.02.001"
+      licence: ["Custom Licence"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - tumor:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - tumor_index:
+      type: file
+      description: BAM/CRAM/SAM index file
+      pattern: "*.{bai,crai,sai}"
+  - intervals:
+      type: file
+      description: bed file containing interval information, optional
+      pattern: "*.{bed}"
+  - fasta:
+      type: file
+      description: Reference genome
+      pattern: "*.{fasta}"
+  - msisensor_baseline:
+      type: file
+      description: File containing custom list of msi regions from a panel of normals to be used as a baseline
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - output_report:
+      type: file
+      description: File containing final report with all detected microsatellites, unstable somatic microsatellites, msi score
+  - output_dis:
+      type: file
+      description: File containing distribution results
```

Comment:
These look like folders not files

Reply:
They are files indeed.

Reply:
Ok, then the stub needs to `touch` the files not `mkdir` them.